apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -41,6 +41,19 @@ fun <T> Expect<T>.isSameAs(expected: T): Expect<T> = _logicAppend { isSameAs(exp
  */
 fun <T> Expect<T>.isNotSameAs(expected: T): Expect<T> = _logicAppend { isNotSameAs(expected) }
 
+/**
+ * Allows to state a reason for one or multiple assertions for a given subject.
+ * @param reason The explanation for the assertion/s.
+ * @param assertionCreator The group of assertions to make.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.14.0
```

Comment:
Could you please provide a sample in addition in AnyAssertionSamples?  thanks
```suggestion
 * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
 *
 * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.AnyAssertionSamples.because
 * @since 0.14.0
```